Resources/skeleton/crud/views/edit.html.twig.twig

```diff
@@ -1,14 +1,17 @@
 {% block extends %}
-{{ "{% extends '::base.html.twig' %}" }}
+{{ "{% extends 'base.html.twig' %}" }}
 {% endblock extends %}
 
 {% block body %}
 {{ "{% block body -%}" }}
     <h1>{{ entity }} edit</h1>
 
-    {{ '{{ form(edit_form) }}' }}
+    {{ '{{ form_start(edit_form) }}' }}
+        {{ '{{ form_widget(edit_form) }}' }}
+        <input type="submit" value="Edit" />
+    {{ '{{ form_end(edit_form) }}' }}
 
-    {% set hide_edit, hide_delete = true, false %}
-    {% include 'crud/views/others/record_actions.html.twig.twig' %}
+{% set hide_edit, hide_delete = true, false %}
+{% include 'crud/views/others/record_actions.html.twig.twig' %}
```

Comment:
Yep: http://twig.sensiolabs.org/doc/functions/include.html

Though I thought that `{%` was deprecated - but it doesn't look to be. So it's really no big deal